doc/src/sphinx/requirements.txt

```diff
@@ -4,7 +4,7 @@ babel==2.9.1
 certifi==2021.5.30
 chardet==4.0.0
 commonmark==0.9.1
-docutils==0.17.1
+docutils==0.16
```

Comment:
this looks like a downgrade

Reply:
I downgraded it because it appears there is a conflict between `sphinx_rtd_theme` and `docutils`, as [shown here](https://stackoverflow.com/a/68023043/6912226). Downgrading docutils seems like the easiest/safest band-aid solution.

Another option is to upgrade Sphinx and the theme as [suggested here](https://stackoverflow.com/a/71069918/6912226), but I haven't tested this:

```
sphinx-rtd-theme==0.5.0  --> sphinx-rtd-theme>=0.5.1
sphinx==3.1.2                      --> sphinx>=4.3.0
```